frontend/src/app/pages/settings-page/settings-page.component.ts

```diff
@@ -42,6 +42,24 @@ export class SettingsPageComponent implements OnInit {
             })
     }
 
+    updateUserDetails() {
+        if(!this.validateUserInput()) return;
+        this.usersService.setUserDetails(this.currentUser.id, this.userName.value, this.userSurname.value)
+            .then(() => {
+              this.currentUser.firstName = this.userName.value;
+              this.currentUser.lastName = this.userSurname.value;
+              sessionStorage.setItem('userInfo', JSON.stringify(this.currentUser));
+              this.snackBar.open("Dane użytkownika zostały zaktualizowane.", "Zamknij", {
```

Comment:
We do not use polish language on our platform :)

Reply:
Got you ;) Noticed error messages in Polish down below in the same file, hence used it here.

Reply:
Thanks for changing it :+1: